wordpress-mobile/WordPress-iOS
# Feature/9854 stock photos free other alert

First step in the implementation of #8954 

We have to start somewhere, so here we go. A short PR, that prepares some ground for the rest of the feature.

In this PR:
- Alert presenting options to navigate to Files or the Free Photo Library
- The alert is not constructed in `AztecPostViewController` but in a separate class. I don't think we should add more code to `AztecPostViewController` unless it is strictly necessary.

To test:
1. Start creating a new post
2. Tap the + button in the lower Format Bar.
3. Tap the ellipsis in the Format Bar
4. There should be an alert with three options (Free photo Library, Files, Cancel)
5. It should still be possible to import content from the Files app.
6. Cancel should dismiss the alert
7. Taping "Free Photo Library" should only dismiss the alert but do nothing else.

![img_2624](https://user-images.githubusercontent.com/2722505/37948643-adc5125a-31c3-11e8-862d-4a0ed84b90c3.PNG)
![img_2625](https://user-images.githubusercontent.com/2722505/37948647-b2154334-31c3-11e8-8cdd-378f39d23f7e.PNG)
